uniflow/op/model/model_config.py

```diff
@@ -61,3 +61,15 @@ class NougatModelConfig(ModelConfig):
     model_name: str = "0.1.0-small"
     batch_size: int = 1
     model_server: str = "NougatModelServer"
+
+@dataclass
+class BedrockModelConfig(ModelConfig):
```

Comment:
I think you can follow `AzureOpenAIModelConfig` config above to not inherit `ModelConfig` while leave required parameter such as `aws_profile`, `aws_region` etc as required argument to fail user early if not specified.

One caveat I found is that if there is required argument in the child ModelConfig class, it will throw error message regarding optional argument ahead of required argument.

Reply:
This is a good point. In the current implementation, if no profile or region is selected, the default profile and region will be applied to create the session. By having a design that allows for early failure in order to handle such cases effectively, especially when user not knowing which region the session is created.